libs/ui/components/surface/project-statistics/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { HStack, Image, VStack } from '@chakra-ui/react';
+import React from 'react';
+import COLORS from '../../../constants/colors';
+import FONTS from '../../../constants/fonts';
+import Text from '../../contents/text';
+import IProjectStatisticsProps from './props';
+
+export default function ProjectStatistics({ nbKeys }: IProjectStatisticsProps) {
```

Comment:
use const Function = () => {} for component

```suggestion
const ProjectStatistics = ({ nbKeys }: IProjectStatisticsProps) => { ... }

export default ProjectStatistics
```

Reply:
ProjectStatistics is the name of the page
ProjectStatisticCard instead
And move it to cards folder